src/main/java/chalmers/pimp/model/canvas/layer/LayerType.java

```diff
@@ -18,5 +18,10 @@
    * The value that represents shape layers. Shape layers aren't pixel-based, instead they use
    * formulas for their representation. Shape layers are considered "smarter" than raster layers.
    */
-  SHAPE
+  SHAPE,
+  /**
+   * The value that represents doodle layers. Doodle layers save points where the user drags the
+   * mouse and then draws lines between the points.
+   */
+  DOODLE
```

Comment:
Why does doodle layers require a unique LayerType value? Aren't doodle layers technically "shapes" (smart objects)?

Reply:
I do not believe shapes and doodle will be manipulated the same way.